packages/console/src/pages/Profile/components/LinkAccountSection/index.tsx

```diff
@@ -42,7 +42,7 @@ function LinkAccountSection({ user, connectors, onUpdate }: Props) {
   const getSocialAuthorizationUri = useCallback(
     async (connectorId: string) => {
       const adminTenantEndpointUrl = new URL(adminTenantEndpoint);
-      const state = buildIdGenerator(8)();
+      const state = generateStandardId(8);
```

Comment:
It seems `generateStandardId` is fixed to 21 digits, and doesn't support customizing the length

Reply:
21 is the default size. `generateStandardId ` takes the size as it's first argument.